sapl/sessao/views.py

```diff
@@ -1762,6 +1779,7 @@ def post(self, request, *args, **kwargs):
                         request.POST['resultado_votacao'])
                     votacao.save()
                 except:
+                    self.logger.error('- Problemas ao salvar registro de votação.')
```

Comment:
Coloquem o except como:
```
except Exception as ex:
```
E daí façam modifiquem o log para:
```suggestion
                    self.logger.error("- Problemas ao salvar registro de votação, {}".format(ex))
```